generator/generator.txt

```diff
@@ -122,6 +117,8 @@ namespace SaintCoinach.Cmd.Commands {
             public string Name { get; set; }
             public string Classes { get; set; }
             public int Level { get; set; }
+            public int ExpReward { get; set; }
```

Comment:
These don't seem to be assigned here. can you include assigning values here or did you assign them manually?

Reply:
They should be assigned on lines 63 and 64?